build/dist

```diff
@@ -248,7 +248,18 @@ echo -e "\nBuilding with..."
 # shellcheck disable=SC2145
 echo -e "\$ ${BUILD_COMMAND[@]}\n"
 
+# shellcheck disable=SC2050
+if [ "$SCALA_VERSION" = "2.12" ]; then
+  EXTRA_BUILD_COMMAND=("$MVN" install $MVN_DIST_OPT $@  -Pscala-2.13 -pl :kyuubi-spark-sql-engine_2.13 )
```

Comment:
`-am` is required, otherwise the mvn command should fail if you don't have .m2 cache

nit: one space between words. EXTRA_BUILD_COMMAND => EXTRA_SPARK_ENGINE_BUILD_COMMAND. `else` is required to safety